data/examples/asn1c_input_tim.bin2xml

```diff
@@ -97,8 +97,8 @@
                     </description>
                 </GeographicalPath>
             </regions>
-            <sspMsgRights1>2</sspMsgRights1>
```

Comment:
Update the README.md how/where these xml examples generated or how they are used?

Reply:
Documentation has been added for which data files are used and where. These don't appear to be referenced anywhere.